libevmasm/Assembly.cpp

```diff
@@ -702,8 +713,8 @@ LinkerObject const& Assembly::assemble() const
 
 
 	if (!m_subs.empty() || !m_data.empty() || !m_auxiliaryData.empty())
-		// Append an INVALID here to help tests find miscompilation.
-		ret.bytecode.push_back(uint8_t(Instruction::INVALID));
+		// Append a BEGINSUB here to help tests find miscompilation.
+		ret.bytecode.push_back(uint8_t(Instruction::BEGINSUB));
```

Comment:
This has to be versioned (evmVersion >= berlin ? beginsub : stop)

Reply:
I thought so, too, but `BEGINSUB` is an invalid opcode prior to berlin...

Reply:
That is true, but one of the outstanding question is whether any deployed contract contains a BEGINSUB opcode between a JUMP/JUMPDEST which would render that JUMP broken.

Iff this PR were to be merged **and** people would deploy contracts with this change, that would increase the number of contracts to be analyzed.

In short, we'd be nicer citizens keeping `INVALID` there prior to Berlin :)